app/views/layouts/_admin_menu.html.haml

```diff
@@ -3,15 +3,15 @@
 %li= link_to 'Admin Dashboard', admin_root_path
 %li= link_to 'View Feedback', admin_feedback_index_path
 %li= link_to 'New Announcement', new_admin_announcement_path
-- if current_user.is_admin_or_organiser?
+- if current_user.admin_or_organiser?
   %li= link_to 'New workshop', new_admin_workshop_path
   %li= link_to 'New sponsor', new_admin_sponsor_path
```

Comment:
@despo I'm wondering what is the point of this check here. I don't see it being necessary, can we remove it? Once you are an admin you should be able to do these actions and we are checking whether the user is an admin when we are including the partial in `app/views/layouts/_aside.html.haml`.

Reply:
Interesting. I wonder if it's a copy paste from the initial partial that was shared between admins and organisers. Yes, let's remove.